src/SearchResults.js

```diff
@@ -44,14 +45,21 @@ export default function Search({
   }, [searchValue, searchQuery]);
 
   return (
-    <SoundList
-      header={
-        showHeader
-          ? `Search results for "${searchQuery || searchValue}"`
-          : undefined
-      }
-      tracks={searchResults}
-      className="py-16"
-    />
+    <>
+      header=
```

Comment:
This part doesn't make sense

Reply:
Could you please elaborate? I don't think I understand

Reply:
what do you expect this line to do?